GameServer/Logic/Packets/Packets.cs

```diff
@@ -1718,11 +1718,11 @@ public RemoveBuff(Unit u, string name) : base(PacketCmdS2C.PKT_S2C_RemoveBuff, u
 
     public class DamageDone : BasePacket
     {
-        public DamageDone(Unit source, Unit target, float amount, DamageType type) : base(PacketCmdS2C.PKT_S2C_DamageDone, target.getNetId())
+        public DamageDone(Unit source, Unit target, float amount, DamageType type, DamageText damageText) : base(PacketCmdS2C.PKT_S2C_DamageDone, target.getNetId())
```

Comment:
Expand this over several lines please, staying under 120 chars per line has a lot of benefits (Just insert a newline before the `:`)